### DISCUSSION TITLE
Hera_setup_intel.csh

### DISCUSSION THREAD

**Discussion by AnningCheng-NOAA** [NCAR/ccpp-scm#514] [Prebuilding, Building, and Compiling] [answered]
where is Hera_setup_intel.csh/Hera_setup_intel.sh now? I have just clone the scm by git clone --recursive https://github.com/NCAR/ccpp-scm, but cannot find the script to configure the environment in hera.

**Comment by hertneky** [comment, marked as answer]
Hi. Actually, we have moved to using modules to set up the environment. Once you clone the code, you can load the module with the following commands (see [section 4.2.3 of the Users Guide](https://ccpp-scm.readthedocs.io/en/v7.0.0/index.html)):

module purge
module use scm/etc/modules
module load hera_intel

Also please note, that the v7.0.0 release was released yesterday - to get that:

git clone --recursive -b v7.0.0 https://github.com/NCAR/ccpp-scm

Please note that some dependencies on mpi may require you to submit your SCM runs on a compute node.

**Comment by sanAkel** [comment]
@hertneky Does the above apply to orion/hercules too? To be specific, can I use [orion_intel.lua](https://github.com/NCAR/ccpp-scm/blob/main/scm/etc/modules/orion_intel.lua)?

**Reply by mkavulich** [reply]
@sanAkel the orion_intel.lua module should work for Orion. If you have a need to use the SCM on Hercules I can look into providing a module for that machine as well, we just haven't used that machine for the SCM yet.

**Reply by sanAkel** [reply]
Thanks @mkavulich, I'll try on orion; if any issues, will get back to you.
